### PR TITLE
Return SubmissionResult containing existing AND missing approvals

### LINKED ISSUE #89: SubmitProposal return missing Approvals
Extend existing API  to include missing-Approval List upon an ApprovalSubmission

### PR DESCRIPTION
**Pull request by NikoBergemann**

### CODE REVIEW

**Review by matthias-geuchen — APPROVED** [review]

File: hlf/chaincode/approval.md [hunk 1]
```diff
@@ -74,7 +74,40 @@ The Errors returned are defined [here](errors.md#Errors).
...
-- None
+### <a id="SubmissionResult" />SubmissionResult
+```json
+{
+  "existingApprovals": {
+    "users": [
+      "EnrollmentID_001"
+    ],
+    "groups": [
+      "Users"
```
Comment (dhelmert) [inline hlf/chaincode/approval.md]:
```suggestion
      "Student"
```

Comment (dhelmert) [reply hlf/chaincode/approval.md]: If im not mistaken, the groups are the same groups are the one in Lagom, which means "Admin", "Lecturer" and "Student".

File: hlf/chaincode/approval.md [hunk 2]
```diff
@@ -74,7 +74,40 @@ The Errors returned are defined [here](errors.md#Errors).
...
+
+### <a id="ApprovalList" />ApprovalList
+```json
+{
+  "users": [
+    "EnrollmentID_001",
+    "EnrollmentID_002"
+  ],
+  "groups": [
+    "Users"
```
Comment (dhelmert) [inline hlf/chaincode/approval.md]:
```suggestion
    "Student"
```

**Review by dhelmert — CHANGES_REQUESTED** [review]

File: hlf/chaincode/approval.md (hunk 1 above)
Comment (NikoBergemann) [reply hlf/chaincode/approval.md]: As of now, the groups are arbitrary, you can put whatever you want into those.
Student is fine though, I'll commit the suggestion

File: hlf/chaincode/approval.md [hunk 3]
```diff
@@ -74,12 +74,45 @@ The Errors returned are defined [here](errors.md#Errors).
 
 ## <a id="Models" />Models
 
-- None
+### <a id="SubmissionResult" />SubmissionResult
+```json
+{
+  "existingApprovals": {
```
Comment (dhelmert) [inline hlf/chaincode/approval.md]:
```suggestion
{
  "operationId" : "0424974c68530290458c8d58674e2637f65abc127057957d7b3acbd24c208f93",
  "existingApprovals": {
```

**Review by dhelmert — APPROVED** [review]

**Review by bastihav — APPROVED** [review]

**Pull request merged**